### PR TITLE
Added the Giveaway Documentation

### PR DESCRIPTION
**Pull request by MrVibesRSA**
Added the giveaway documentation, This is my first ever pull request  so I hope I done it right.

### CODE REVIEW

File: extensions/giveaways/Howto.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# How to install, setup and working with the commands.
+
+### Installing the actions.
```
Comment (GoWMan813) [inline extensions/giveaways/Howto.md:20]: Would put Import code here using markup.

File: extensions/giveaways/Importcode.md [hunk 2]
```diff
@@ -0,0 +1 @@
...L/So0gEDD+OnO70Cu6NeORBRBDRgVPDXAKZ3XEI8imbJPJ29CgPO/l/YJkx59tSCbEr7LX3QsqjK/vLXZAzwea106ekFKedlV2YrJXHabWK25J0YP7Eyazj2NdceiNUiJC0ICqwf51Ia6WNfUO9DkYyXRujK6ImfxJMO14ppgwLl2hORpF1mNBL5JAwpphy77odKfGmkrs4uNBtLcaZ7Jss2Zs3LulFtZr9ZLlSJmZmFizjdNgi6Nrk1s7tZBOomxst0n4D9rcH7yIKffqZ5T/iDwASB4hIxB4JECp4TBX7AkJauF2rNPPbeDNfev623YpBSuqg4HMQu6cRm/kifXnr6h+8L4lFmc6uoBPOyqWcMXKgImBu4nucQ/Arz2i2m1y3mOHsCDjsnrSajbK4S2Q+2FSs56RQVzbcIuOKIFXunoOPnd9FcDZ45gktJEWBiYL4PNENUGA9J4hEE8JYJl7uKeu/sNPfL3Vtz8iCT1MjSbTe54CBaGEVJkiZjTTLGPvsVgLJh21tgysW+4GLztHr2hCqN3ZC6AhnHbvu0x75DSOYixanddeH76r1Rr7+XGO2SYiqedHEnQ/+FKNZ7LSdjZezR4rkPVOvIoggjiilv7WifkF+IiV18RdjRdoMPCWyiij02n4bYl5Ix9c5bVoK95JeFBLV8btsNV13wkyFnyOcMZnhudworx6BQutB/X/pgfu/bZV7qcvHxmTWMp4SS7wIk1DWECxSkBzDQkCDBaIAMIzBVh9T43suJomJR4qmoOVs5+LQ7DaCYll/kwHKWy5GcBoRAaEGc/Igpwg0SISOu40ifSLp345/laF/UA/z8AO2/kkRcLghOf8dYAnyDwAkF9sjBQIXYEUzxsJyg+3tXkkorSB9s8agMa/5UZqf+9EoUQxyjPa05cqRNc3VdADAAlZFHvCAMDA8MeYUbfksU8ybnk+w6nGTnCaI2Z+P5s0NdAH1MKYFU4NmzQ+0piEJw5HvM5470CMfvnEzwokjjpcfovJoCfeIonWfq2zpO5zmOVBIjJMYM1NbFiIKjBh8GUQp3HeYIHQYBeQVH+oan6uyX0JHOOFZtF4nJ1Lakp/0ind/MRRQDBpKTxXwz5fOJI3TsPkvmEYUcimFWxWEATpBTZKTUgcYcK/+diZUyA/ACT/8yH2P/ZE9mjqJU9FMu1+L5f/wf/EHRqqiFAAA=
```
Comment (GoWMan813) [inline extensions/giveaways/Importcode.md]: No need for file if code moved.

File: extensions/giveaways/Howto.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# How to install, setup and working with the commands.
```
Comment (GoWMan813) [inline extensions/giveaways/Howto.md:18]: Add readme contents above this line.

File: extensions/giveaways/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Giveaway-Extention
```
Comment (GoWMan813) [inline extensions/giveaways/README.md]: No need of this file if contents moved.

**Review by GoWMan813 — CHANGES_REQUESTED** [review]
Looks good but need it all to be in one page if possible.  I made notes on how to do that.  It is mainly for rendering but will make sense soon.

File: extensions/giveaways/Howto.md (hunk 1 above)
Comment (MrVibesRSA) [reply extensions/giveaways/Howto.md:20]: Ok no problem I'll get on that for you.

**Pull request merged**